### ISSUE TITLE
Drop dropdowns in info.ts

### ISSUE THREAD

**Issue by JRegimbal** [Shared-Reality-Lab/IMAGE-browser#168]
The point of the dropdowns was to add a way to go through a large number of renderings quickly. However,
1. We have fewer renderings now that many have been consolidated.
2. Almost every screen reader allows skipping between headers.

At this point the experience would probably be more consistent and simpler by removing dropdowns. Assigning to @Sabrina-Knappe since she was already thinking about this.

**Comment by jeffbl** [comment]
@Sabrina-Knappe I'm talking wtih @Cybernide and if we want to do this, it needs to go to @jaydeepsingh25 asap so it can be implemented. Is there more design work to be done on this, or is it still uncertain whether this change is desired?

**Comment by Sabrina-Knappe** [comment]
The design work is finished. I need to do one last check with NVDA, and @Cybernide needs to check with JAWS, but after that it can be implemented. Here is sample example of how it would work and how the html should be generated. The basic idea is that each overall section would be described under an H1, and then for the default audio rendering, the whole rendering as well as the segments would be individual buttons with play/pause functionality. The buttons are nested within H2 so that they can be found easily using header jumping. Should be minimal clicks with both VO and NVDA.
[UI Tests.zip](https://github.com/Shared-Reality-Lab/IMAGE-browser/files/8144959/UI.Tests.zip)
.

**Comment by jeffbl** [comment]
At this point, we won't have time to iterate this before CSUN even if it goes for implementation today. Really remote chance of this going in for CSUN if not in and stable by Friday.

**Comment by Sabrina-Knappe** [comment]
I see no issue with pushing this post CSUN, especially considering how close the deadline is. This fix mostly affects VO users, who are a relatively small portion of our audience anyway.

From: Jeff Blum ***@***.***>
Date: Monday, February 28, 2022 at 2:09 PM
To: Shared-Reality-Lab/IMAGE-browser ***@***.***>
Cc: Sabrina Knappe ***@***.***>, Mention ***@***.***>
Subject: Re: [Shared-Reality-Lab/IMAGE-browser] Drop dropdowns in info.ts (Issue #168)

At this point, we won't have time to iterate this before CSUN even if it goes for implementation today. Really remote chance of this going in for CSUN if not in and stable by Friday.

**Comment by jeffbl** [comment]
@Cybernide fyi postponing this based on @Sabrina-Knappe comment above.

**Comment by Cybernide** [comment]
Yes, it's unfortunate, but has to happen - there are a few more critical issues user facing

**Comment by jeffbl** [comment]
Start with @jaydeepsingh25 , and he can reassign as necessary.

**Comment by jeffbl** [comment]
@Sabrina-Knappe Is this ready for implementation? If so, it needs to be assigned to @jaydeepsingh25 with necessary wireframes. If not, please comment with remaining work until this can be assigned to development.

**Comment by Sabrina-Knappe** [comment]
Assigned to Jaydeep for implementation.

**Comment by jeffbl** [comment]
Assigning to @Sabrina-Knappe since there is a PR raised on this that is assigned to her for review before this goes into production.

**Comment by jeffbl** [comment]
Pending panel testing. Moving forward one sprint.

**Comment by Cybernide** [comment]
@Sabrina-Knappe Let me know when you're ready to panel test

**Comment by Cybernide** [comment]
If I understand right, this is pending some logic fixes

**Comment by jeffbl** [comment]
Moving to Oct3, still pending user feedback.

**Comment by Sabrina-Knappe** [comment]
Updates from User-Facing:
David liked the changes! He is having difficulty opening the extension at all because he finds the context menu hard to manipulate. We need to check with a few more users ( #264 ) to determine whether this is a recurring problem, but if users like the new UI once they access it, we will likely merge this change.

**Comment by Sabrina-Knappe** [comment]
@Cyan Kuo, ***@***.***> Did you have a chance to check this on JAWS? If yes, then I’ll re-assign to Jaydeep with appropriate wireframes. Although the previously attached zip file is probably more useful for him than wireframes.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by Cybernide** [comment]
This has been JAWS tested, and while I am personally not a fan, we have two users who strongly prefer this UI change. Their opinion takes precedent over mine.
